src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/crash-analysis-group-filter-your-crashes.mdx

```diff
@@ -186,21 +186,19 @@ Resolved crashes include a banner identifying who resolved the crash and when. B
     title="Examine crash report details: Export to Xcode or resymbolicate your source code."
   >
     From the **Crash types** table, you can drill down into a specific [crash type](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/investigate-mobile-app-crash-report). From here you can:
-
-    * Examine the [interaction trail](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/interactions-page) from session start through the crash event.
```

Comment:
Has this feature been removed or just moved in the Mobile UI?

Reply:
We removed this specific component. In the event trail (below the stack trace) a user can click on the Interaction icon and limit the event trail to interactions, which gives the same information as the interaction trail used to give.